fuzz/cmp.c

```diff
@@ -155,6 +155,19 @@ static int process_pollReq(OSSL_CMP_SRV_CTX *srv_ctx,
     return 0;
 }
 
+static int reset_transaction(OSSL_CMP_SRV_CTX *srv_ctx)
+{
+    ERR_raise(ERR_LIB_CMP, CMP_R_ERROR_PROCESSING_MESSAGE);
+    return 0;
+}
+
+static OSSL_CMP_PKISI *initiate_delayed_delivery(OSSL_CMP_SRV_CTX *srv_ctx,
+                                                 const OSSL_CMP_MSG *req)
+{
+    ERR_raise(ERR_LIB_CMP, CMP_R_ERROR_PROCESSING_MESSAGE);
+    return NULL;
+}
+
```

Comment:
Just a note:
Meanwhile I doubt if we need to define all the other dummy request handling callback functions in this file
(because if the function pointers are `NULL` anyway an error will be thrown),
but what you did is at least somewhat consistent with what was done before.